bff/fancy.py

```diff
@@ -162,6 +162,39 @@ def get_peaks(s: pd.Series, distance_scale: float = 0.04):
     return peaks_dates.values, peaks[1]['peak_heights']
 
 
+def idict(d: Dict[Any, Hashable]) -> Dict[Hashable, Any]:
+    """
+    Invert a dictionary meaning that keys will be become values and values will become keys.
```

Comment:
What happens if a value is present multiple times? Since keys must be unique in a dictionary, which one is lost? An explanation and an example would be nice.